core/modules/parsers/parseutils.js

```diff
@@ -385,4 +385,35 @@ exports.parseAttribute = function(source,pos) {
 	return node;
 };
 
+/*
+Given a parsed attribute node, serialize it back into its source string representation.
+*/
+exports.serializeAttribute = function(node) {
+	if(!node || typeof node !== 'object' || !node.name || !node.type) {
```

Comment:
TW prefers double quotes for strings

Reply:
Updated.

Note that there are many `let` and `'` in #8494 I saw when adding new tests. Haven't we put eslint in CI?